### DISCUSSION TITLE
Add custom text and - perhaps - images?

### DISCUSSION THREAD

**Discussion by veitch123** [Anze/KeyCluCask#88] [Ideas]
I'd love to use keyclu as a repository for all sorts of tips I can write to myself. There are things I need to remember in apps that aren't just shortcuts.

**Comment by Anze** [comment]
Hey [veitch123](https://github.com/veitch123),

Thanks for your suggestion! The feature you’re asking for isn’t something the app is designed to do. However, you can achieve the same behavior by adding custom text (with or without hotkeys) globally or per application using `My Shortcuts` section in `Settings`. While images aren't supported, you can use emojis as an alternative.


<img width="1436" alt="Screenshot 2024-08-05 at 13 50 05" src="https://github.com/user-attachments/assets/531463de-3577-406c-b036-a7e22cf9cd2d">

**Comment by veitch123** [comment]
Hey thanks so much for this. 
A few problems; 

1.  I have to add hotkeys otherwise the text doesn't show up. 
2. The text always shows up last (I'd prefer my shortcuts and text to be prioritised.)
3. There's no way to add paragraph breaks etc. 
![CleanShot 2024-09-10 at 16 54 37](https://github.com/user-attachments/assets/30c2d1cf-57a5-4e2b-a6de-6c6ebbc2f940)

**Comment by Anze** [comment]
Hey [veitch123](https://github.com/veitch123),

For the first issue, you can change the settings to show inactive shortcuts, as custom shortcuts without an assigned key are treated as inactive. This behavior may change in the future.

Regarding the second issue, I believe you're referring to custom shortcuts always appearing in the last column. I'm currently working on making them blend into the full list of shortcuts. Are you suggesting they should appear at the beginning of the list or in the first column?

The third issue is a bit more complex, as the expectation was for a short description rather than full notes. Could you please reach out to me via email at sms[@]tatarenkov.name so we can discuss a proper solution?
